### PR TITLE
Final styles

### PR DESCRIPTION
**Pull request by Shedrack-Sunday**
## Changes Made:

- [ ] User icon Added

- [ ] Style components to fit WireFrame

- [ ] CSS styles

## View:
yt

### CODE REVIEW

File: src/components/Books.js [hunk 1]
```diff
@@ -12,23 +11,18 @@ const Container = styled.div`
...
-        <h1>Add new Book!</h1>
-      )}
-
+      {books.map((book) => (
+        <CardBook
+          key={book.item_id}
+          id={book.item_id}
+          name={book.title}
+          author="Shedrack Sunday"
+          category={books.category}
```
Comment (zainsadaqat) [inline src/components/Books.js:23]: It says, The prop `category` is marked as required in `CardBook`, but its value is `undefined`. Kindly fix this error.

Screenshot 2022-11-12 at 1 36 58 PM

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Required Changes 🔁
# Hello @Shedrack-Sunday  😄 

You're doing a great job on your project. We're proud of you. Good job so far! 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
  ✅  Correct use of Github Flow
  ✅  No linter errors
  ✅  Descriptive Pull Request

## Required Changes ♻️

- [ ] Missing Design. Kindly make sure it matches the required design 👍🏻 

[YOUR DESIGN]
Screenshot 2022-11-12 at 1 51 57 PM

[REQUIRED DESIGN]
Screenshot 2022-11-12 at 1 52 11 PM

- [ ] Missing Category name in the design.

[YOUR DESIGN]
Screenshot 2022-11-12 at 1 53 31 PM

[REQUIRED DESIGN]
Screenshot 2022-11-12 at 1 54 24 PM


_Check the comments under the review._




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Books.js (hunk 1 above)
Comment (BertrandConxy) [inline src/components/Books.js:23]: - [ ] as requested by the previous reviewer, please update your styles to match the design given. Look at the comparison below 👇 
**Your design**

**Given design**


HINT:
- Please rotate the progress bar to match the one on the design 😉

File: src/components/CardBook.js [hunk 2]
```diff
@@ -104,12 +103,17 @@ const CardBook = (props) => {
...
         </div>
         <div>
           <Button>Comments</Button>
-          <Button type="button" onClick={() => dispatch(removeBook(id))}>
+          <Button
+            type="button"
+            onClick={() => {
+              dispatch(removeBook(id));
+            }}
+          >
```
Comment (BertrandConxy) [inline src/components/CardBook.js:116]: - [ ] Please add the author as requested by the design given. Please look at the comparison below 👇 
**Your design**

**Given design**

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @Shedrack-Sunday, 

Good job so far! 👏 👏 
There are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 🏃‍♂️ 

## Highlights 💪 
- No linter errors
- correct flow is used
- good PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Books.js (hunk 1 above)
Comment (Shedrack-Sunday) [reply src/components/Books.js:23]: This is supposed to be dynamic. Why must it have to be exact? The requirement does not state so. @BertrandConxy

File: src/components/CardBook.js (hunk 2 above)
Comment (Shedrack-Sunday) [reply src/components/CardBook.js:116]: Check my pull request. It's working. You probably need to run it properly. Take caution with details, please.

File: src/components/Books.js (hunk 1 above)
Comment (Shedrack-Sunday) [reply src/components/Books.js:23]: This is it from my machine working perfectly. please update all your dependencies. the error is from your end.
yt

**Review by Munsa1 — APPROVED** [review]
Hi @Shedrack-Sunday, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**